src/Helper.js

```diff
@@ -0,0 +1,20 @@
+export function calculateWinner(squares) {
+	const lines = [
+		[0, 1, 2],
+		[3, 4, 5],
+		[6, 7, 8],
+		[0, 3, 6],
+		[1, 4, 7],
+		[2, 5, 8],
+		[0, 4, 8],
+		[2, 4, 6],
+	];
+	for (let i = 0; i < lines.length; i++) {
+		const [a, b, c] = lines[i];
+		//console.log(squares[i]+"here")
+		if (squares[a] && squares[a] === squares[b] && squares[a] === squares[c]) {
+			return squares[b];		
+		}
+	}
+	return null;
+}
```

Comment:
Please leave a line at the end of your file. If you look to this notation, you'll find `No new line at EOF`